jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/DekorateUtil.java

```diff
@@ -13,21 +13,19 @@
  */
 package org.eclipse.jkube.kit.common.util;
 
-import org.apache.maven.project.MavenProject;
 
-public class ResourceMojoUtil {
+import org.eclipse.jkube.kit.common.JavaProject;
+
+public class DekorateUtil {
 
     public static final String DEFAULT_RESOURCE_LOCATION = "META-INF/jkube";
-    private static final String[] DEKORATE_CLASSES = new String[]{
-            "io.dekorate.annotation.Dekorate"
-    };
+    public static final String DEKORATE_GROUP = "io.dekorate";
 
-    private ResourceMojoUtil() {
+    private DekorateUtil() {
     }
 
-    public static boolean useDekorate(MavenProject project) {
-        return new ProjectClassLoaders(MavenUtil.getCompileClassLoader(project))
-                .isClassInCompileClasspath(true, DEKORATE_CLASSES);
+    public static boolean useDekorate(JavaProject project) {
+      return JKubeProjectUtil.hasDependencyWithGroupId(project, DEKORATE_GROUP);
```

Comment:
Since we are changing the behavior and I'm not sure if we have an E2E or IT for Dekorate, please make sure that the Quickstart still works after the change

Reply:
I tested on spring-boot-dekorate quickstart and it seems to be working